src/sidebar/util/time.js

```diff
@@ -1,57 +1,13 @@
 'use strict';
 
-const minute = 60;
-const hour = minute * 60;
-
-function lessThanThirtySecondsAgo(date, now) {
-  return now - date < 30 * 1000;
-}
-
-function lessThanOneMinuteAgo(date, now) {
-  return now - date < 60 * 1000;
-}
-
-function lessThanOneHourAgo(date, now) {
-  return now - date < 60 * 60 * 1000;
-}
-
-function lessThanOneDayAgo(date, now) {
-  return now - date < 24 * 60 * 60 * 1000;
-}
-
-function thisYear(date, now) {
-  return date.getFullYear() === now.getFullYear();
-}
-
-function delta(date, now) {
-  return Math.round((now - date) / 1000);
-}
-
-function nSec(date, now) {
```

Comment:
This function starts a collection of several string-formatting functions. They have been moved around such that they are all grouped together.